apps/dotcom/client/src/tla/components/TlaFileShareMenu/file-share-menu.module.css

```diff
@@ -0,0 +1,151 @@
+/* ------------------- Share menu ------------------- */
+
+.shareMenu {
+	width: 240px;
+	--share-menu-padding: 12px;
+}
+
+.helpButton {
+	all: unset;
+	position: absolute;
+	top: 0px;
+	right: 0px;
+	width: 40px;
+	height: 40px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	color: var(--tla-color-text-3);
+	cursor: pointer;
+}
+
+.helpButton[data-active='true'] {
+	color: var(--tla-color-text-1);
+}
+
+@media (hover: hover) {
+	.helpButton:hover {
+		color: var(--tla-color-text-1);
+	}
+}
+
+.content {
+	position: relative;
+}
+
+.section {
+	display: flex;
+	padding: var(--share-menu-padding);
+	flex-direction: column;
+	gap: var(--share-menu-padding);
+}
+
+.section:nth-of-type(n + 2) {
+	border-top: 1px solid var(--tla-color-border);
+}
+
+.controlGroup {
```

Comment:
can i just say, that one of the things i like about css modules is getting away from the BEM stuff?

Reply:
Yeah I like it too!